backend/config/channel_layers.py

```
@@ -0,0 +1,170 @@
+    # Prefer a real Redis PING when the client library is installed.
+    try:
+        import redis
+
+        client = redis.from_url(url, socket_connect_timeout=timeout, socket_timeout=timeout)
+        try:
+            return bool(client.ping())
+        finally:
+            try:
+                client.close()
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)